### PR TITLE
[DOC-6183] Add clarifying language about Helm support

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-6183]

## Previews
[v21.1/deploy-cockroachdb-with-kubernetes-insecure.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v21.1/deploy-cockroachdb-with-kubernetes-insecure.html)
[v21.1/deploy-cockroachdb-with-kubernetes.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v21.1/deploy-cockroachdb-with-kubernetes.html)
[v21.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v21.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.html)
[v21.1/orchestrate-a-local-cluster-with-kubernetes.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v21.1/orchestrate-a-local-cluster-with-kubernetes.html)

[v21.2/deploy-cockroachdb-with-kubernetes-insecure.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v21.2/deploy-cockroachdb-with-kubernetes-insecure.html)
[v21.2/deploy-cockroachdb-with-kubernetes.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v21.2/deploy-cockroachdb-with-kubernetes.html)
[v21.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v21.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.html)
[v21.2/orchestrate-a-local-cluster-with-kubernetes.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v21.2/orchestrate-a-local-cluster-with-kubernetes.html)

[v22.1/deploy-cockroachdb-with-kubernetes.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v22.1/deploy-cockroachdb-with-kubernetes.html)
[v22.1/deploy-cockroachdb-with-kubernetes-insecure.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v22.1/deploy-cockroachdb-with-kubernetes-insecure.html)
[v22.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v22.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.html)
[v22.1/orchestrate-a-local-cluster-with-kubernetes.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v22.1/orchestrate-a-local-cluster-with-kubernetes.html)

[v22.2/deploy-cockroachdb-with-kubernetes.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v22.2/deploy-cockroachdb-with-kubernetes.html)
[v22.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v22.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.html)
[v22.2/orchestrate-a-local-cluster-with-kubernetes.md](https://deploy-preview-15664--cockroachdb-docs.netlify.app/docs/v22.2/orchestrate-a-local-cluster-with-kubernetes.html)

## Tests
- [x] Local build
- [x] Linkcheck

### CODE REVIEW

File: _includes/v21.1/orchestration/kubernetes-limitations.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ The CockroachDB Helm chart requires Helm 3.0 or higher. If you attempt to use an
 Error: UPGRADE FAILED: template: cockroachdb/templates/tests/client.yaml:6:14: executing "cockroachdb/templates/tests/client.yaml" at <.Values.networkPolicy.enabled>: nil pointer evaluating interface {}.enabled
 ~~~
 
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. 
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm Chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm Chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (mdlinville) [inline _includes/v21.1/orchestration/kubernetes-limitations.md:19]:
```suggestion
The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```

File: _includes/v21.1/orchestration/start-cockroachdb-helm-secure.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
-{{site.data.alerts.callout_danger}}
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm Chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm Chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (mdlinville) [inline _includes/v21.1/orchestration/start-cockroachdb-helm-secure.md:3]:
```suggestion
The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```

File: _includes/v21.2/orchestration/kubernetes-limitations.md [hunk 3]
```diff
@@ -14,7 +14,9 @@ The CockroachDB Helm chart requires Helm 3.0 or higher. If you attempt to use an
 Error: UPGRADE FAILED: template: cockroachdb/templates/tests/client.yaml:6:14: executing "cockroachdb/templates/tests/client.yaml" at <.Values.networkPolicy.enabled>: nil pointer evaluating interface {}.enabled
 ~~~
 
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned.  
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm Chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm Chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs.
```
Comment (mdlinville) [inline _includes/v21.2/orchestration/kubernetes-limitations.md:19]:
```suggestion
The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs.
```

File: _includes/v21.2/orchestration/start-cockroachdb-helm-secure.md [hunk 4]
```diff
@@ -1,7 +1,9 @@
-{{site.data.alerts.callout_danger}}
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm Chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm Chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (mdlinville) [inline _includes/v21.2/orchestration/start-cockroachdb-helm-secure.md:3]:
```suggestion
The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```

File: _includes/v22.1/orchestration/kubernetes-limitations.md [hunk 5]
```diff
@@ -14,7 +14,9 @@ The CockroachDB Helm chart requires Helm 3.0 or higher. If you attempt to use an
 Error: UPGRADE FAILED: template: cockroachdb/templates/tests/client.yaml:6:14: executing "cockroachdb/templates/tests/client.yaml" at <.Values.networkPolicy.enabled>: nil pointer evaluating interface {}.enabled
 ~~~
 
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. 
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm Chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm Chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (mdlinville) [inline _includes/v22.1/orchestration/kubernetes-limitations.md:19]:
```suggestion
The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```

File: _includes/v22.1/orchestration/start-cockroachdb-helm-secure.md [hunk 6]
```diff
@@ -1,7 +1,9 @@
-{{site.data.alerts.callout_danger}}
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm Chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm Chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (mdlinville) [inline _includes/v22.1/orchestration/start-cockroachdb-helm-secure.md:3]:
```suggestion
The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```

File: _includes/v22.2/orchestration/kubernetes-limitations.md [hunk 7]
```diff
@@ -14,7 +14,9 @@ The CockroachDB Helm chart requires Helm 3.0 or higher. If you attempt to use an
 Error: UPGRADE FAILED: template: cockroachdb/templates/tests/client.yaml:6:14: executing "cockroachdb/templates/tests/client.yaml" at <.Values.networkPolicy.enabled>: nil pointer evaluating interface {}.enabled
 ~~~
 
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. 
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm Chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm Chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (mdlinville) [inline _includes/v22.2/orchestration/kubernetes-limitations.md:19]:
```suggestion
The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```

File: _includes/v22.2/orchestration/start-cockroachdb-helm-secure.md [hunk 8]
```diff
@@ -1,7 +1,9 @@
-{{site.data.alerts.callout_danger}}
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm Chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm Chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (mdlinville) [inline _includes/v22.2/orchestration/start-cockroachdb-helm-secure.md:3]:
```suggestion
The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```

File: _includes/v21.1/orchestration/kubernetes-limitations.md [hunk 9]
```diff
@@ -14,7 +14,9 @@ The CockroachDB Helm chart requires Helm 3.0 or higher. If you attempt to use an
 Error: UPGRADE FAILED: template: cockroachdb/templates/tests/client.yaml:6:14: executing "cockroachdb/templates/tests/client.yaml" at <.Values.networkPolicy.enabled>: nil pointer evaluating interface {}.enabled
 ~~~
 
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. 
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (towfiqa) [inline _includes/v21.1/orchestration/kubernetes-limitations.md:19]: Aren't we repeating ourselves here? We say no new features are planned. Then with the However we add another no new feature development is currently planned.

Comment (towfiqa) [reply _includes/v21.1/orchestration/kubernetes-limitations.md:19]: Also it is missing the verbiage around deprecation announced 6 months earlier

File: _includes/v21.1/orchestration/start-cockroachdb-helm-secure.md [hunk 10]
```diff
@@ -1,7 +1,9 @@
-{{site.data.alerts.callout_danger}}
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (towfiqa) [inline _includes/v21.1/orchestration/start-cockroachdb-helm-secure.md:3]: Same issues as previously commented

File: _includes/v21.2/orchestration/kubernetes-limitations.md [hunk 11]
```diff
@@ -14,7 +14,9 @@ The CockroachDB Helm chart requires Helm 3.0 or higher. If you attempt to use an
 Error: UPGRADE FAILED: template: cockroachdb/templates/tests/client.yaml:6:14: executing "cockroachdb/templates/tests/client.yaml" at <.Values.networkPolicy.enabled>: nil pointer evaluating interface {}.enabled
 ~~~
 
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned.  
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs.
```
Comment (towfiqa) [inline _includes/v21.2/orchestration/kubernetes-limitations.md:19]: This should add the wording around deprecation

File: _includes/v21.2/orchestration/start-cockroachdb-helm-secure.md [hunk 12]
```diff
@@ -1,7 +1,9 @@
-{{site.data.alerts.callout_danger}}
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (towfiqa) [inline _includes/v21.2/orchestration/start-cockroachdb-helm-secure.md:3]: Same two issues commented previously

File: _includes/v22.1/orchestration/kubernetes-limitations.md [hunk 13]
```diff
@@ -14,7 +14,9 @@ The CockroachDB Helm chart requires Helm 3.0 or higher. If you attempt to use an
 Error: UPGRADE FAILED: template: cockroachdb/templates/tests/client.yaml:6:14: executing "cockroachdb/templates/tests/client.yaml" at <.Values.networkPolicy.enabled>: nil pointer evaluating interface {}.enabled
 ~~~
 
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. 
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (towfiqa) [inline _includes/v22.1/orchestration/kubernetes-limitations.md:19]: same problem as others

File: _includes/v22.1/orchestration/start-cockroachdb-helm-secure.md [hunk 14]
```diff
@@ -1,7 +1,9 @@
-{{site.data.alerts.callout_danger}}
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (towfiqa) [inline _includes/v22.1/orchestration/start-cockroachdb-helm-secure.md:3]: same problem as others

File: _includes/v22.2/orchestration/kubernetes-limitations.md [hunk 15]
```diff
@@ -14,7 +14,9 @@ The CockroachDB Helm chart requires Helm 3.0 or higher. If you attempt to use an
 Error: UPGRADE FAILED: template: cockroachdb/templates/tests/client.yaml:6:14: executing "cockroachdb/templates/tests/client.yaml" at <.Values.networkPolicy.enabled>: nil pointer evaluating interface {}.enabled
 ~~~
 
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. 
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (towfiqa) [inline _includes/v22.2/orchestration/kubernetes-limitations.md:19]: same problems as others

File: _includes/v22.2/orchestration/start-cockroachdb-helm-secure.md [hunk 16]
```diff
@@ -1,7 +1,9 @@
-{{site.data.alerts.callout_danger}}
-The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
+The CockroachDB Helm chart is compatible with Kubernetes versions 1.22 and earlier.
+
+The CockroachDB Helm chart is currently not under active development, and no new features are planned. However, Cockroach Labs remains committed to fully supporting the Helm chart by addressing defects, providing security patches, and addressing breaking changes due to deprecations in Kubernetes APIs. (the latest version as of this writing). However, no new feature development is currently planned. If you are experiencing issues with a Helm deployment on production, contact our [Support team](https://support.cockroachlabs.com/).
```
Comment (towfiqa) [inline _includes/v22.2/orchestration/start-cockroachdb-helm-secure.md:3]: same problem as others

**Review by towfiqa — CHANGES_REQUESTED** [review]
Please see comments.

File: _includes/v21.1/orchestration/kubernetes-limitations.md (hunk 9 above)
Comment (mdlinville) [reply _includes/v21.1/orchestration/kubernetes-limitations.md]: I'm not sure what happened here -- I replaced the old text with --- the old text? I'm not sure. It's fixed now.

**Review by udnay — APPROVED** [review]
LGTM

**Pull request merged**